### ISSUE TITLE
Basic style formatting for extracts and figure captions

### ISSUE THREAD

**Issue by hlesaint** [openmindanesth/openmind#82]
It would be great to have a basic style formatting for extracts and figure captions.

**Important:** subscript, superscript (because it's often used in science)
**Helpful:** bold, italic (to emphasize and visually structure)
**why not:** underline, bullets (open for debate)

**Comment by tgetgood** [comment]
We could take in arbitrary markdown and render it in the extracts easily enough. 

Do we want to allow things like headings, bullet lists, etc.? Extracts will get substantially longer and start to look more like mini-papers. I don't know if that's good or bad, but it is a change in direction.

**Comment by hlesaint** [comment]
Yes, you're right. I think it makes sense to keep the limit to about one sentence. No bullets and headings for now.

**Comment by hlesaint** [comment]
What do you think about including links? This would give some additional degrees of freedom, although I'm currently not sure if it would help or if it could get too wild. Sometimes it makes sense to keep a structure limited, which gives a common ground of usage.
